timely_beliefs/visualization/utils.py

```diff
@@ -28,7 +32,8 @@ def plot(
     :param ci: The confidence interval to highlight in the time series graph
     :param intuitive_forecast_horizon: If true, horizons are shown with respect to event start rather than knowledge time
     :param interpolate: If True, the time series chart shows a user-friendly interpolated line rather than more accurate stripes indicating average values
-    :param plottable_df: Optionally, specify as plottable DataFrame directly together with a sensor name, a sensor unit and a y-axis value range for the event values (if None, we create it)
+    :param plottable_df: Optionally, specify as plottable DataFrame directly together with a sensor name,
```

Comment:
Is this in place of the bdf param? I mean, are we using the bdf param at all if we pass in plottable_df? If not, we'd need only one argument, which can either be a BeliefsDataFrame, or this tuple of things.

Reply:
I removed this parameter altogether. It was originally created as a temporary and hacky speed improvement for a demo.